### PR TITLE
Bump aiohttp version

### PR DESCRIPTION
**Pull request by TAG-Epic**
## Summary

#257 
## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [x] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Review by VincentRPS — APPROVED** [review]
lgtm

**Comment by veni-vidi-code** [comment]
should be tested first

**Comment by RPSMain** [comment]
i guess i will through the next couple of days test this on nextcord and multiple other libs
https://github.com/DisnakeDev/disnake/pull/164
https://github.com/iDevision/enhanced-discord.py/pull/120
probably 3.8.1 and 4.0.0 using everything webhooks ext's etc

File: requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
-aiohttp>=3.6.0,<3.8.0
+aiohttp>=3.6.0,<4.0.0
```
Comment (RPSMain) [inline requirements.txt:1]: Would be better to bump every new update to make sure users aren't using non working versions
```suggestion
aiohttp>=3.6.0,<3.8.1
```

**Review by RPSMain — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (ooliver1) [reply requirements.txt:1]: semver means that there won't be breaking changes before the next major

Comment (RPSMain) [reply requirements.txt:1]: changed it to 3.9

**Comment by ooliver1** [comment]
tested ```py
      Successfully uninstalled aiohttp-3.6.3
Successfully installed aiohttp-3.8.1 aiosignal-1.2.0 async-timeout-4.0.1 charset-normalizer-2.0.9 frozenlist-1.2.0```

**Pull request merged**